app/src/main/res/layout/activity_options.xml

```diff
@@ -83,6 +84,13 @@
 			android:layout_weight="2" />
 	</LinearLayout>
 
+	<androidx.appcompat.widget.SwitchCompat
+		android:id="@+id/optionsOrderZeroItemsLast"
+		style="@style/Theme.ShoppingList.SwitchCompat"
+		android:onClick="onClickOrderZeroItemsLastSwitch"
+		android:text="@string/optionsOrderZeroItemsLast"
+		android:textSize="@dimen/optionTextSize"
+		app:layout_constraintTop_toBottomOf="@id/optionsOrderDropdown" />
```

Comment:
> This view is not constrained horizontally

As with the one at the very top we can ignore the warning (or constrain it to start and end of parent).
```suggestion
		app:layout_constraintTop_toBottomOf="@id/optionsOrderDropdown"
		tools:ignore="MissingConstraints" />
t" />
```

Reply:
ho right, good catch, I forgot to run the linter!